contracts/gov/user-keeper/GovUserKeeper.sol

```diff
@@ -659,6 +659,19 @@ contract GovUserKeeper is IGovUserKeeper, OwnableUpgradeable, ERC721HolderUpgrad
         return _getFreeAssets(voter, false, lockedProposals, unlockedNfts);
     }
 
+    function getDelegatedAssetsAmount(
+        address delegator,
+        address delegatee
+    ) external view override returns (uint256) {
+        return _usersInfo[delegator].delegatedTokens[delegatee];
+    }
+
+    function getMicropoolTotalStakeAmount(
```

Comment:
Please use `tokenBalance` function for this

Reply:
done